start-site/src/main/resources/application.yml

```diff
@@ -181,6 +181,13 @@ initializr:
             version: 23.2.15
           - compatibilityRange: "[2.7.0-M1,2.8.0-M1)"
             version: 23.3.5
+      hilla:
+        groupId: dev.hilla
+        artifactId: hilla-bom
+        versionProperty: hilla.version
+        mappings:
+          - compatibilityRange: "[2.7.0-M1,2.8.0-M1)"
```

Comment:
Unlike the range below (`[2.6.0-M1,3.1.0-M1)`), this range doesn't cover Spring Boot 3.0. I think another mapping is needed for the bom that's compatible with Spring Boot 3.

Reply:
ah, indeed.. by that time, hilla 2.0 was not in maven.. i will update it.